classes/quiz_settings.php

```diff
@@ -267,6 +267,7 @@ private function compute_config_key() {
     private function compute_config() {
         switch ($this->get('requiresafeexambrowser')) {
             case settings_provider::USE_SEB_NO:
+                $this->set('templateid', 0);
```

Comment:
@nhoobin  what about create a new method process_no_seb_config() and move this line there? it will be consistent with others and will give some flexibility in the future if we'd like to do some extra stuff there

Reply:
Good point. I'll add that now.